client/packages/coldchain/src/Monitoring/ListView/TemperatureLog/Toolbar.tsx

```diff
@@ -44,15 +44,15 @@ export const Toolbar: FC<{ filter: FilterController }> = () => {
                   name: t('label.from-datetime'),
                   urlParameter: 'datetime',
                   range: 'from',
-                  maxDate: new Date(),
+                  maxDate: DateUtils.addMinutes(new Date(), -30),
```

Comment:
I wonder if we would ever work with countries that cross multiple timezones - and they WOULD want to look at times in the 'future'

Reply:
oh, good point! this is where being conservative could come back to bite me!